config/event-schema.json

```diff
@@ -49,6 +52,24 @@
     },
     "user_id": {
       "type": "integer"
+    },
+    "data": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
+          "required": ["key", "value"]
+        }
+      ]
+    },
+    "event_tags": {
+      "anyOf": [
+        {
+          "type": "array",
+          "items" : [ { "type": "string" } ]
+        },
+        { "type": "null" }
+      ]
     }
   },
```

Comment:
Would it be worth adding ["additionalProperties": false](http://json-schema.org/understanding-json-schema/reference/object.html?highlight=additional#additionalproperties) to ensure we don't consider valid any properties that aren't listed here?